dotcom-rendering/src/web/layouts/StandardLayout.tsx

```diff
@@ -761,6 +761,8 @@ export const StandardLayout = ({ CAPIArticle, NAV, format }: Props) => {
 												CAPIArticle.pageType
 													.isPaidContent
 											}
+											sectionName={CAPIArticle.pillar}
```

Comment:
Could we replace `CAPIArticle.pillar` with `format`, please 🙏 `CAPIArticle.pillar` is a legacy property which although not quite deprecated it could soon be. We now use `format` when making decisions based on the pillar of an article.